thunder/core/baseutils.py

```diff
@@ -75,6 +75,10 @@ class TensorProxyInterface:
     pass
 
 
+class TorchAutogradFunctionXtxProxyInterface:
```

Comment:
Typo here, should be `TorchAutogradFunctionCtxProxyInterface`

Reply:
Great catch, thank you Luca! opened 1147